test/infrastructure/docker/Makefile

```diff
@@ -89,7 +90,7 @@ test-e2e: ## Run the end-to-end tests
 		$(MAKE) run-e2e
 
 run-e2e:
-	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=20m -tags=e2e
+	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=20m -tags=e2e -e2e.config="$(abspath e2e/e2e.conf)"
```

Comment:
Will this respect the variables that are set if called from `test-e2e`? I suspect there might be issues if the env vars are used to overwrite the name/registry of the CAPD docker image and they do not match the value that is hard coded in the config file.

Reply:
Hi @detiber,

Yep. That's a conversation @chuckha and I are having in the PR's comments section.